### ISSUE TITLE
The new ditto implemented features include cetane as part of the feature set

### ISSUE THREAD

**Issue by hgromer** [ECRL/ECNet#26]
Is this intended? When running the limit input feature task for example, cetane is included as one of the possible features.

**Comment by tjkessler** [comment]
Would
```python
item_collection.add_attribute(Attribute('CN (Exp)', is_descriptor=False)
```
work? Right now I'm omitting the is_descriptor argument, and it looks like it defaults to True

**Comment by hgromer** [comment]
Yes, that would prevent that attribute from being included in any algorithm calculations.

**Comment by tjkessler** [comment]
Fixed as of 8fef3d1
